components/camera/client.go

```diff
@@ -131,7 +140,7 @@ func (c *client) Read(ctx context.Context) (image.Image, func(), error) {
 		return nil, nil, err
 	}
 
-	if resp.MimeType != expectedType {
+	if expectedType != "" && resp.MimeType != expectedType {
```

Comment:
This prevents log spam when the user expressed no preference for any specific MimeType

Reply:
I think so